hubblestack_nova/stat.yaml

```diff
@@ -0,0 +1,123 @@
+stat:
+  grub_conf:
+    data:
+      CentOS_Linux_6:
+        - '/etc/grub.conf':
+            tag: ['CIS-1.5.1', 'CIS-1.5.2']
+            mode: 600
+            user: 0
+            group: 0
+      CentOS_Linux_7:
+        - '/etc/grub2/grub.cfg':
+            tag: ['CIS-1.5.1', 'CIS-1.5.2']
```

Comment:
Should this be a single tag (string) or a list of tags? I expect there will only be one tag per file for stat, so it will probably be just a string, not a list.